src/components/Sponsors/Sponsors.js

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+import React from "react";
+import "./Sponsors.css";
 
 function Sponsors() {
   return (
-    <div>Hi Maritza!</div>
+    <div class="sponsors-container">
+      <div class="sponsors-title">
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">P</div>
+          <div class="letter">O</div>
+          <div class="letter">N</div>
+        </div>
+        <div class="word">
+          <div class="letter">S</div>
+          <div class="letter">O</div>
+          <div class="letter">R</div>
+          <div class="letter">S</div>
+        </div>
```

Comment:
Why did you break this up into two? Aren't the letters supposed to be more separated?

Reply:
I did this so when the screen minimizes the word splits into two parts instead of a column of individual letters. Let me know if I should change it.

Reply:
Hmm... okay, I understand. I would recommend (instead of splitting the words into two) using `vw` to dynamically change the block sizes based on the current viewport window.